doc/mgr/orchestrator_cli.rst

```diff
@@ -105,20 +118,43 @@ filtered to a particular node:
 
 ::
 
-    ceph orchestrator device ls [--host=...] [--devname=...] [--refresh]
+    ceph orchestrator device ls [--host=...] [--refresh]
 
 Create OSDs
 ^^^^^^^^^^^
 
 Create OSDs on a group of devices on a single host::
 
-    ceph orchestrator osd create <host> <ceph-volume-invocation…>
+    ceph orchestrator osd create [flags...] <host>:<drive>
+    ceph orchestrator osd create -i <path-to-drive-group.json>
 
-See :ref:`ceph-volume-invocation… <ceph-volume-overview>` for details. E.g.
-``ceph orchestrator osd create host1 lvm create …``
 
 The output of ``osd create`` is not specified and may vary between orchestrator backends.
 
+Where ``drive.group.json`` is a JSON file containing the following fields:
+
+* ``host_pattern``: An fnmatch pattern to select hosts. Can also be a single host.
```

Comment:
It'd be nice to be able to use the host pattern on the CLI with the non-JSON invocation of `osd create`, and likewise, be able to use the JSON invocation for the drive spec with the host pattern specified on the CLI. Not a big deal, but I could see it being useful.

Reply:
let's tackle this after the merge of @jmolmo 's PR